packages/k/kitty/pspec_x86_64.xml

```diff
@@ -705,12 +697,41 @@ Kitty is designed from the ground up to support all modern terminal features, su
         </Files>
     </Package>
     <History>
-        <Update release="55">
-            <Date>2023-08-20</Date>
-            <Version>0.29.2</Version>
-            <Comment>Packaging update</Comment>
-            <Name>Reilly Brogan</Name>
-            <Email>solus@reillybrogan.com</Email>
+        <Update release="56">
+            <Date>2023-09-24</Date>
+            <Version>0.30.0</Version>
+            <Comment>kitty: Update to 0.30.0
```

Comment:
I just pushed a change to the solbuild package that should prevent it from doing history generation during rebuilds. Can you please rebuild the package one more time and then amend your commit to include the fixed pspec_x86_64.xml file? You will need to do this from a fully updated unstable system to have the most recently updated solbuild.